application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -215,22 +219,20 @@ public void onButtonClick(@NotNull ButtonInteractionEvent event, @NotNull List<S
         long callerId = Long.parseLong(args.get(2));
         long interactorId = event.getMember().getIdLong();
 
-        if (callerId == interactorId) {
-            int currentPage = Integer.parseInt(args.get(3));
-            int turnPageBy = Integer.parseInt(args.get(4));
-
-            long guildId = Long.parseLong(args.get(0));
-            long targetId = Long.parseLong(args.get(1));
-            int pageToDisplay = currentPage + turnPageBy;
-
-            event
-                .editMessage(
-                        auditUser(guildId, targetId, interactorId, pageToDisplay, event.getJDA()))
-                .queue();
-        } else {
+        if (callerId != interactorId) {
             event.reply("Only the user who triggered the command can use these buttons.")
                 .setEphemeral(true)
                 .queue();
         }
```

Comment:
This has to go all the way to the top of the method and `return;` then.

Reply:
ohh yea i forgot sry